### PR TITLE
Docs in preparation for 0.10.0 release

### PR DESCRIPTION
**Pull request by oicheryl**
Updates the versions across various pages
Deprecate storageos.feature.* label prefix for storageos.com/
Adds GUI docs

### CODE REVIEW

File: _docs/reference/gui.md [hunk 1]
```diff
@@ -7,4 +7,20 @@ module: reference/gui
 
 # Graphical user interface (GUI)
 
-This feature is coming soon.
+StorageOS provides a GUI for cluster and volume management.
+
+**Note**: This feature is currently in alpha. To access the GUI from version 0.10.0,
+set the environment variable `ALPHA_FEATURES=true`.
```
Comment (croomes) [inline _docs/reference/gui.md]: `ALPHA_FEATURES=true` was only needed for 0.9.  In 0.10 it's enabled by default.  Maybe change to say that it's in beta?

File: assets/Vagrantfile [hunk 2]
```diff
@@ -3,6 +3,9 @@ $install_storageos = <<SCRIPT
 # Prerequisites
 sudo modprobe nbd nbds_max=1024
 
+# Activate UI
+export ALPHA_FEATURES=true
+
```
Comment (croomes) [inline assets/Vagrantfile]: Doesn't hurt to add this, but not needed.

File: _docs/reference/gui.md (hunk 1 above)
Comment (oicheryl) [reply _docs/reference/gui.md]: Done.

File: assets/Vagrantfile (hunk 2 above)
Comment (oicheryl) [reply assets/Vagrantfile]: Reverted.

File: _docs/manage/features/placement.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ module: manage/features/placement
 # Volume Placement
 
 Master volume placement can be influenced when the volume created by specifying the requested master node name in the
-`storageos.hint.master` label.  When possible, the scheduled will place the master volume on this node.  If this is not
+`storageos.com/hint.master` label.  When possible, the scheduled will place the master volume on this node.  If this is not
```
Comment (croomes) [inline _docs/manage/features/placement.md]: scheduler

**Pull request merged**